app/models/member.rb

```diff
@@ -0,0 +1,6 @@
+class Member < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
```

Comment:
- [ ] Inside Model user.rb:
-  Please you need to enable `:registerable`, `:database_authenticable`, `:recoverable` the code will look like this but not exactly the same : 
  `devise :database_authenticatable, :registerable,
	      :recoverable, :rememberable, :validatable, :confirmable`
- 
![Capture](https://user-images.githubusercontent.com/63972670/180322392-addcdc05-e06a-41a0-8784-cf395e7115bd.JPG)
- From the request above: The application allows for registering a new user, hashed password is stored in the database,  user can reset the password in the application

Reply:
Hello @OLIPLICHE Thanks for the hints. You said like this but not exactly same. In your hint you mentioned everything that devise has and said not exactly same. So should I add some more extra things besides what you showed ?

Reply:
Yes, Please you can add. I used "not exactly the same" to give the opportunity to freely write the code as you want. just make sure it works based on the requirements of the project.